tests/no_std.rs

```diff
@@ -0,0 +1,27 @@
+#![no_std]
+#![allow(dead_code)]
+
+use auto_impl::auto_impl;
+
+mod core {}
```

Comment:
Is the idea of including these at the root here just to make sure we don't accidentally use module instead of crate paths to the standard library crates?

Reply:
yep, that was the idea